types/types.go

```diff
@@ -50,6 +50,11 @@ const (
 	LonghornLabelInstanceManagerType = "instance-manager-type"
 	LonghornLabelVolume              = "longhornvolume"
 
+	KubernetesDeprecatedRegionLabelKey = "failure-domain.beta.kubernetes.io/region"
+	KubernetesDeprecatedZoneLabelKey   = "failure-domain.beta.kubernetes.io/zone"
+	KubernetesNewRegionLabelKey        = "topology.kubernetes.io/region"
```

Comment:
Better not to use the word `New` here. New is relative. What if this tag becomes deprecated later?

We can call it `TopologyLabel` vs `FailureDomainLabel`.